ql/src/java/org/apache/hadoop/hive/ql/security/authorization/command/CommandAuthorizerV2.java

```diff
@@ -98,7 +98,7 @@ private static void addPermanentFunctionEntities(SessionState ss, List<ReadEntit
   }
 
   private static List<HivePrivilegeObject> getHivePrivObjects(List<? extends Entity> privObjects,
-      Map<String, List<String>> tableName2Cols, HiveOperationType hiveOpType) throws HiveException {
+      Map<String, List<String>> tableName2Cols, HiveOperationType hiveOpType, BaseSemanticAnalyzer sem) throws HiveException {
     List<HivePrivilegeObject> hivePrivobjs = new ArrayList<HivePrivilegeObject>();
     if (privObjects == null){
```

Comment:
nit: If I was editing this method - I would maybe consider making the format uniform within in it.
I.E. it seems to have at least 3 different coding styles.
if (conditional){
if(conditional){
if (conditional) {

Reply:
Sure.